curator-test-zk34/README.md

```diff
@@ -0,0 +1,11 @@
+# ZooKeeper 3.4 Compatibility
+
+Apache Curator 4.0 adds best-effort backward compatibility for ZooKeeper 3.4 ensembles and the ZooKeeper 3.4 library/JAR. This module, `curator-test-zk34`, exists to run the standard Curator tests using ZooKeeper 3.4. It does this via Maven. In the curator-test-zk34 pom.xml:
+
+- The Curator modules framework and recipes libraries are included - both main and test JARs - but the ZooKeeper dependency is excluded (otherwise ZooKeeper 3.5.x would be brought in)
+- The curator-test module is included but as version 2.12.0 which brings in ZooKeeper 3.4.8
```

Comment:
I'm confused why this is necessary if we're excluding the transitive ZK version anyway?